ui/app/templates/components/clients/history.hbs

```diff
@@ -0,0 +1,96 @@
+            Tracking is currently disabled and data is not being collected. Historical data can be searched, but you will need to <LinkTo @route="vault.cluster.clients.edit">edit the configuration</LinkTo> to enable tracking again.
+            </AlertBanner>
+        {{/if}}
+        <h1 data-test-client-count-title class="title is-4 has-bottom-margin-s">Monthly history</h1>
+        <p class="has-bottom-margin-s">This data is presented by full month. If there is data missing, it's possible that tracking was turned off at the time. Vault will only show data for contiguous blocks of time during which tracking was on.</p>
+        <PricingMetricsDates
+        @queryStart={{@model.queryStart}}
+        @queryEnd={{@model.queryEnd}}
+        @resultStart={{@model.activity.startTime}}
+        @resultEnd={{@model.activity.endTime}}
+        @defaultSpan={{or @model.config.defaultReportMonths 12}}
+        @retentionMonths={{@model.config.retentionMonths}}
+        />
+    {{/if}}
+    {{#unless this.hasClientData}}
+        {{#if (eq @tab "current")}}
+            {{#if (eq @model.config.enabled 'On')}}
+                <EmptyState @title="No data received" @message="Tracking is turned on and Vault is gathering data. It should appear here within 30 minutes." />
+            {{/if}}
+        {{else}}
+                <EmptyState @title="No data received" @message="No data exists for that query period. Try searching again." />
+        {{/if}}
+    {{else}}
+        <div class="card has-bottom-margin-m">
+            <div class="card-content">
+            <div class="is-flex is-flex-center">
+                <div class="is-flex-1">
+                <h2 class="title is-5 is-marginless">Total usage</h2>
+                <p class="sub-text">These totals are within this namespace and all its children.</p>
+                </div>
+                <LearnLink @path="/tutorials/vault/usage-metrics">
+                Learn more
+                </LearnLink>
+            </div>
+            <hr>
+            <div class="columns">
+                <div class="column" data-test-client-count-stats>
+                    <StatText @label="Total active Clients" @value={{or @model.activity.clients @model.activity.total.clients}} @size="l" 
+                        @subText="The sum of unique entities and active direct tokens; Vault's primary billing metric."/>
+                    </div>
+                    <div class="column">
+                    <StatText class="column" @label="Unique entities" @value={{or @model.activity.distinct_entities @model.activity.total.distinct_entities}} @size="l" 
+                        @subText="Representation of a particular user, client or application that created a token via login."/>
+                    </div>
+                    <div class="column">
+                    <StatText class="column" @label="Active direct tokens" @value={{or @model.activity.non_entity_tokens @model.activity.total.non_entity_tokens}} @size="l" 
```

Comment:
similar tabbing issues, just do a double check. You've got some elements that are nested but not tabbed in this file.